src/nvim/ui.h

```diff
@@ -13,14 +13,22 @@ typedef enum {
   kUIPopupmenu,
   kUITabline,
   kUIWildmenu,
+  kUIWidgetCount,
```

Comment:
won't this make `switch` blocks weird, if the compiler insists that this is handled as a `case`?

Reply:
Yes, but can't that be changed if we ever need a switch statement? (which we don't if the keep using the existing pattern of specific widget code always calling into the generic one, and never the other way around)